app/pages/project/talk-images.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import getSubjectLocation from '../../lib/get-subject-location.coffee';
+import TalkStatus from './talk-status';
+import VideoPlayer from '../../components/video-player';
+
+export default class TalkImages extends React.Component {
+
+  render() {
+    return (
+      <div className="project-home-page__container">
+
+        {this.props.images.map((image) => {
+          const subject = getSubjectLocation(image);
```

Comment:
I think you can use the `Thumbnail` component (`app/components/thumbnail.jsx`) instead of the code here. That way, we don't have duplicated code for previewing subjects.